### ISSUE TITLE
Would it be possible to run in quiet mode?

### ISSUE THREAD

**Issue by ramou** [c3g/genpipes_in_a_container#4]
Right now I'm using this for boutiques. Various informational things pop up while the container loads and it would be nice to be able to suppress that. 
```
2019/02/25 17:34:43.17 parrot_run[314] notice: parrot_run 6.2.8 FINAL from source has not been tested on Linux 4.9.0-7-amd64 yet, this may not work
LibCvmfs version 2.4, revision 25
2019/02/25 17:34:44.24 parrot_run[324] notice: parrot_run 6.2.8 FINAL from source has not been tested on Linux 4.9.0-7-amd64 yet, this may not work

Wait while Genpipes module are loaded. This could take a while,
  especially if the cvmfs cache is new

LibCvmfs version 2.4, revision 25
ERROR: Unable to locate a modulefile for 'mugqic/genpipes/3.1.3'
```

**Comment by ramou** [comment]
After hunting for another problem I found that parrot can take `-d clear` as a flag to remove its frequent debug messages.

**Comment by poquirion** [comment]
fixed by #9
